cmapi/mcs_cluster_tool/cluster_app.py

```diff
@@ -157,29 +158,31 @@ def stop(
         # TODO: investigate more on how changing the hardcoded timeout
         #       could affect put_config (helpers.py broadcast_config) operation
         timeout = 0
-    _ = ClusterHandler.shutdown(logger=logger, in_transaction=True)
+
+    resp = client.shutdown_cluster({'in_transaction': True})
```

Comment:
How much time does the shutdown_cluster wait till it returns?

Reply:
Does it take user-specified graceful timeouts into account or this logic is expressed in a different manner?

Reply:
Fixed after discussing. Added timeouts.